src/Symfony/Component/HttpKernel/Tests/HttpKernelTest.php

```diff
@@ -112,7 +113,7 @@ public function testHandleHttpException()
     /**
      * @dataProvider getStatusCodes
```

Comment:
The recommended way is indeed to add `@group legacy`, but because I have added a new test that is testing the same outcome but a different way,  I wanted the test methods to read the same (apart from the `testLegacy` prefix on the old test method), I think this improves readability but happy to change if the core team agree with you